### PR TITLE
Improve localization

### PR DESCRIPTION
**Pull request by shindgewongxj**

### CODE REVIEW

File: Cork/Localizable.xcstrings [hunk 1]
```diff
@@ -19834,7 +19984,7 @@
         "zh-Hans" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "有可用软件包更新"
```
Comment (Jerry23011) [inline Cork/Localizable.xcstrings:19837]: Please keep these, 有 is the term used by Apple.

**Review by Jerry23011 — CHANGES_REQUESTED** [review]

File: Cork/Localizable.xcstrings (hunk 1 above)
Comment (shindgewongxj) [reply Cork/Localizable.xcstrings:19837]: 🫡  Sorry for the mistakes

**Comment by Jerry23011** [comment]
Please look at this commit (https://github.com/buresdv/Cork/pull/373/commits/ba04766a6b27b78f14caa9d16bbaddbb7d5e11cf)

`\n` means a line break. You can use ⌥ ↩ to insert a line break.

**Comment by Jerry23011** [comment]
Since Cork is an open-source software, I'm using 你 instead of 您（what would be usually used in closed-source commercial apps) https://github.com/buresdv/Cork/pull/373/commits/b199e77fd29049c9468ef25ea7e7c8fe7897f24c

拷贝 is the right term for copy in Chinese, Windows has been misleading us for decades by translating it into 复制 (duplicate), thanks a lot Microsoft. :)

**Review by Jerry23011 — APPROVED** [review]
LGTM

Love your translations with JSON parsing, couldn't do so well if it were me translating. 🫶

**Comment by Jerry23011** [comment]
感谢！@shindgewongxj

**Comment by shindgewongxj** [comment]
> 感谢！@shindgewongxj

非常感谢你帮助我完善和修正这些被我忽略掉的细节，将会在后续的本地化中注意这一部分！🫡 再次感谢！@Jerry23011

**Comment by buresdv** [comment]
我的兄弟们，你们做的饭菜真好吃 🫡

**Pull request merged**